docs: Remove remaining Deno content from Contributing Guide (#243)
pulsate-dev/pulsate: CONTRIBUTING.md

  - If you want to submit a PR in progress, [please submit it as "Draft"](https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
- Pull request title conforms to [Conventional Commits](#commit-message)

> Then run `pnpm format` to format the code. (If you are using VSCode, the formatting will be done automatically.)

Reviewer: ```suggestion
After finishing, please run `pnpm format` to format the code. (If you are using VSCode, it will be formatted automatically.)
```
ちょっと違和感を覚えたので提案しますが, 日本語の英訳としてはこのような感じでどうでしょうか

Author: Fixed https://github.com/pulsate-dev/pulsate/pull/243/commits/cf9360eb6cc5c78cc467dcf777cffba8e49b78ff
